docs/index.html

```diff
@@ -2,4 +2,22 @@
 title: Google Analytics 4 Implementation record
 ---
 <h1 class="govuk-heading-l">{{ page.title }}</h1>
-<p class="govuk-body">This site contains a record of the data implementation for Google Analytics 4 on GOV.UK.</p>
+<p class="govuk-body">This site contains a record of the data implementation for Google Analytics 4 on GOV.UK Publishing. It has been built as an aid to developers on GOV.UK Publishing but is being made available for anyone who may find our approach useful.</p>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-one-half">
+    <h2 class="govuk-heading-l govuk-!-font-weight-bold">Data</h2>
+    <p class="govuk-body">Details of the data pushed to the dataLayer when tracking occurs, including event and attribute details.</p>
+    <p class="govuk-body govuk-!-margin-bottom-0">
+      <a href="{{ site.baseurl }}/data" class="govuk-link govuk-!-font-weight-bold">Browse data</a>
```

Comment:
This link is broken for me

Reply:
Ah yes, forgot to update when I restructured the file layout. Fixed now.